src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -197,15 +177,15 @@ void AsyncRPCOperation_sendmany::main() {
     LogPrintf("%s",s);
 }
 
-bool IsFromAnyTaddr(const PaymentSource& paymentSource) {
+bool IsFromAnyTaddr(const ZTXOSelector& ztxoSelector) {
```

Comment:
I misinterpreted this name the first time I reviewed it, as meaning "the source of funds is some t-addr". It actually means "the source of funds can be any arbitrary t-addr". Consider changing the name.